packages/beacon-node/src/network/core/networkCoreWorkerHandler.ts

```diff
@@ -107,7 +107,10 @@ export class WorkerNetworkCore implements INetworkCore {
       loggerOpts: modules.logger.toOpts(),
     };
 
-    const worker = new Worker("./networkCoreWorker.js", {workerData} as ConstructorParameters<typeof Worker>[1]);
+    const worker = new Worker("./networkCoreWorker.js", {
+      workerData,
+      resourceLimits: {maxYoungGenerationSizeMb: 152},
```

Comment:
Can you add a comment here linking / describing why 152?

Reply:
might be worth exposing as a hidden cli flag so we don't need to PR to test other values

Reply:
Updated.